doc/source/ray-libraries.rst

```diff
@@ -143,3 +149,7 @@ spaCy is a library for advanced Natural Language Processing in Python and Cython
 .. |spacy| image:: images/spacy.png
     :class: inline-figure
     :height: 30
+    
+.. |nlu| image:: images/nlu.png
```

Comment:
Btw I noticed the logo has a bunch of whitespace, it would be better to trim that before uploading.